src/database/migrations/versions/2da0dc469be8_temp_migration.py

```
@@ -5,48 +5,67 @@
 Create Date: 2025-01-07 21:36:44.563306
 
 """
+
 from typing import Sequence, Union
 
 from alembic import op
 import sqlalchemy as sa
 
 
 # revision identifiers, used by Alembic.
-revision: str = '2da0dc469be8'
-down_revision: Union[str, None] = '32b1054a69e3'
+revision: str = "2da0dc469be8"
+down_revision: Union[str, None] = "32b1054a69e3"
 branch_labels: Union[str, Sequence[str], None] = None
 depends_on: Union[str, Sequence[str], None] = None
 
 
 def upgrade() -> None:
     # ### commands auto generated by Alembic - please adjust! ###
-    op.alter_column('activation_tokens', 'token',
-               existing_type=sa.VARCHAR(length=255),
-               type_=sa.String(length=64),
-               existing_nullable=False)
-    op.alter_column('password_reset_tokens', 'token',
-               existing_type=sa.VARCHAR(length=255),
-               type_=sa.String(length=64),
-               existing_nullable=False)
-    op.alter_column('refresh_tokens', 'token',
-               existing_type=sa.VARCHAR(length=255),
-               type_=sa.String(length=64),
-               existing_nullable=False)
+    op.alter_column(
+        "activation_tokens",
+        "token",
+        existing_type=sa.VARCHAR(length=255),
+        type_=sa.String(length=64),
+        existing_nullable=False,
+    )
+    op.alter_column(
+        "password_reset_tokens",
+        "token",
+        existing_type=sa.VARCHAR(length=255),
+        type_=sa.String(length=64),
+        existing_nullable=False,
+    )
+    op.alter_column(
+        "refresh_tokens",
+        "token",
+        existing_type=sa.VARCHAR(length=255),
+        type_=sa.String(length=64),
+        existing_nullable=False,
+    )
```

This upgrade block shortens token columns from 255 to 64 characters. Reducing maximum length can silently truncate existing token values and cause broken authentication/session flows. Before running this migration in a non-empty DB you should either:

- Validate that no existing token exceeds 64 characters and fail early if any do, or
- Provide a careful data migration that truncates or re-generates tokens, or
- Add an explicit migration step: create new narrower column, copy/transform data with checks, then swap columns.

The initial migration created these columns with length=255, so truncation is a real risk (see initial migration) . Ensure you test this migration against a database snapshot/staging environment.